src/app/component/event-card/event-card.component.ts

```diff
@@ -132,12 +136,17 @@ export class EventCardComponent {
   getImageUrls(): RegExpMatchArray | null | undefined {
     const urlRegex = /https:.*?\.(?:png|jpg|svg|gif|jpeg|webp)/gi;
     const imgArray = this.event?.content.match(urlRegex);
+    this.imageUrls = imgArray
     return imgArray;
   }
 
+  clickToLoadMedia(){
+    this.amIFollowingTheAuthor = true;
```

Comment:
This is incorrect data representation.  Instead of creating another variable to store the state of 'showed' or 'not showed'. we are reusing a different state variable.

This will cause problems when we edit the code further in future.